src/KintoID.sol

```diff
@@ -405,13 +416,23 @@ contract KintoID is
      */
     function removeSanction(address _account, uint16 _countryId) public override onlyRole(KYC_PROVIDER_ROLE) {
         if (balanceOf(_account) == 0) revert KYCRequired();
+
+        // Check if account is in protection period (10 days from last sanction)
```

Comment:
Is this check needed? I do not think we need to touch remove sanction. I think it should only apply to addSanction

Reply:
It is needed because otherwise you can cancel and immediately add sanction, which allows to perma block users.